docs/upgrade/1.8.2_to_2.0.0.rst

```diff
@@ -1,19 +1,18 @@
-Upgrade from 1.7.0 to 1.7.1
+Upgrade from 1.8.2 to 2.0.0
```

Comment:
This is git's move detection being off and treating a new file as an edit. :/

Reply:
Split into its own commit as fae248d27de83b4432f27e3fcfd33d7a54b72010 to aid diff readability.